### PR TITLE
[Logo] Logo language change

### PR DESCRIPTION
**Pull request by MichaelCherry**
# Description

Adding logos that start with FR text and logic to toggle between original and new depending on the current language

## Type of change

- New feature (non-breaking change which adds functionality)

### Main logo:
En:
image

Fr:
image

### In-test logo:
En:
image

Fr:
image

# Testing

Check that toggling language toggles the logos

# Checklist

Applicable for all code changes.

~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
- [x] My changes generate no new compiler warnings
- [x] My changes generate no new accessibility errors and/or warnings
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
- [x] I have researched WCAG2.1 accessibility standards and met them in this PR (can be navigated using a keyboard)
- [x] My changes look good on IE 11+ and Chrome

### CODE REVIEW

File: frontend/src/text_resources.js [hunk 1]
```diff
@@ -347,10 +347,10 @@ let LOCALIZE = new LocalizedStrings({
         orgChartInstructions:
           "Below is a tree view of the organization chart. Once selected, you can use the arrow keys to navigation, expand, and collapse information.",
         backgroundInformation: {
-          title: "Background Information"
+          title: "Overview"
```
Comment (fnormand01) [inline frontend/src/text_resources.js]: Are these @caleybrock's changes from one of the last PRs?
Did you merge in master?

File: frontend/src/text_resources.js [hunk 2]
```diff
@@ -347,10 +347,10 @@ let LOCALIZE = new LocalizedStrings({
         orgChartInstructions:
           "Below is a tree view of the organization chart. Once selected, you can use the arrow keys to navigation, expand, and collapse information.",
         backgroundInformation: {
-          title: "Background Information"
```
Comment (caleybrock) [inline frontend/src/text_resources.js]: Why are these changed? Maybe merge in master?

Comment (MichaelCherry) [reply frontend/src/text_resources.js]: merged in master: 29179ff (that was weird)

File: frontend/src/text_resources.js (hunk 1 above)
Comment (MichaelCherry) [reply frontend/src/text_resources.js]: merged in master: 29179ff (that was weird)
